tensorflow/compiler/mlir/tosa/transforms/strip_quant_types.cc

```diff
@@ -1,27 +1,26 @@
-/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Don't update this as part of a functional CL. It makes it difficult to track the scale of functional.